### DISCUSSION TITLE
Changing Sort Methods for Folders

### DISCUSSION THREAD

**Discussion by Rage-GitHub** [diced/zipline#542] [Ideas]
The idea of being able to sort folders based on file data would be a huge plus in my book. 

Ideally, it would be nice to be able to sort folders, either by a default value or persistent specified value for that folder, with a date, size, name, or custom sort order.

**Comment by diced** [comment]
thanks for suggesting, but it already exists in v4!

**Reply by Rage-GitHub** [reply]
Ah, great! Much appreciated then!
